src/libraries/System.Private.CoreLib/src/System/Environment.Browser.cs

```diff
@@ -34,5 +35,35 @@ private static OperatingSystem GetOSVersion()
         /// </summary>
         /// <returns>Path of the executable that started the currently executing process</returns>
         private static string? GetProcessPath() => null;
+
+        /// <summary>
+        /// Get the CPU usage, including the process time spent running the application code, the process time spent running the operating system code,
+        /// and the total time spent running both the application and operating system code.
+        /// </summary>
+        [SupportedOSPlatform("maccatalyst")]
```

Comment:
These is `Environment.UnixOrBrowser.cs`. More the implementation into that file to avoid the duplication